recipients-resolver/src/test/java/com/redhat/cloud/notifications/recipients/startup/StartupUtilsTest.java

```diff
@@ -25,7 +25,7 @@ public class StartupUtilsTest {
 
     @Test
     void readKeystoreTest() {
-        String resourceName = "testKeystore.jks";
+        String resourceName = "testKeystore.p12";
```

Comment:
We also need to add `quarkus.rest-client.it-s2s.key-store-type=P12` in `application.properties` file ([JKS is the defaut type](https://quarkus.io/guides/all-config#quarkus-rest-client-config_quarkus-rest-client-client-key-store-type))

Reply:
Replacing config in `application.properties` with parameter/env var to allow for testing with the existing JKS in ephemeral. See [!153239](https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/153239) for more context.